client/src/components/PeopleDetailPanel.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { TextField, Button, Grid } from "@material-ui/core";
+import EmploymentHistoryTable from "./EmployeementHistoryTable";
+
+export interface DetailPanelState {
+    value: string;
+}
+
+export interface DetailPanelProps {
+    personId: number;
+    personComment: string;
+}
+
+export default class PeopleDetailPanel extends React.Component<DetailPanelProps, DetailPanelState> {
+    
+    state = {
+        value: this.props.personComment
+    }
+
+    handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        this.setState( {value: event.target.value} );
+    };
+
+
+    render() {
+        return (
```

Comment:
This render looks like it needs some refactoring. I wrote this, but I would request @MountBaqer to review it first.

Reply:
Yup you are right, I merged the branch for demo purposes, I'll "unmerge" it so we can do through a pull request to milestone-8